CMakeLists.txt

```diff
@@ -15,8 +15,11 @@ set(
   "Semicolon separated list of sanitizer names, e.g., 'address;leak'. \
 Supported sanitizers are address, leak, undefined and thread."
 )
+option(BUILD_CLI "Build the 'lightbgm' command-line interface in addition to lib_lightgbm" ON)
```

Comment:
Note that the new options `BUILD_CLI` and `INSTALL_HEADERS` both default to `ON` so that this PR preserves the current default behavior on `master`.